### PR TITLE
Add role with magento preload 

### PR DESCRIPTION
**Pull request by Szpadel**
Require https://github.com/mageops/packages-rpm/pull/2 to me merged first

### CODE REVIEW

**Comment by Szpadel** [comment]
@pinkeen this is now ready to be merged

File: roles/cs.magento-optimize-iops/templates/magento-preload.service.j2 [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Preload magento small PHP files into memory
```
Comment (krisdante) [inline roles/cs.magento-optimize-iops/templates/magento-preload.service.j2]: Magento

File: site.step-45-app-deploy.yml [hunk 2]
```diff
@@ -17,6 +17,11 @@
         deploy_release_id: "{{ ansible_date_time.iso8601_basic_short }}"
         deploy_keep_current_release: "{{ deploy_skip_installation | default(false) and deploy_previous_release_exists }}"
         deploy_previous_release_id: "{{ deploy_previous_release_exists | ternary((_current_release_check.stat.lnk_target | default('')).split('/') | last, false) }}"
+    
+    - name: Skip instalation
```
Comment (krisdante) [inline site.step-45-app-deploy.yml]: Use the previous release of code, skip installation

File: site.step-45-app-deploy.yml [hunk 3]
```diff
@@ -68,6 +74,9 @@
           role: cs.magento-configure
           tasks_from: 200-static-symlink.yml
 
+    - role: cs.magento-optimize-iops
+      when: aws_use
```
Comment (krisdante) [inline site.step-45-app-deploy.yml]: Please add some additional variable to turn this on or off.

**Pull request merged**